content/participants/ttjc@konnster99/index.md

```diff
@@ -21,20 +21,20 @@ I want to learn.
 
 ## something interesting about you?
 
-Well I like like to watch movies and vibe with the punjabi music.
+Well I like to watch movies and vibe with the punjabi music.
 
 ### help I need?
 
-I would be needing a lot of help because I literslly will be learning js formthe scratch in this 1 month.
+I would be needing a lot of help because I literally will be learning js form the scratch in this 1 month.
```

Comment:
from* scratch

Reply:
oohk i'm really sorry for the typos.
Will update